bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BMap.java

```diff
@@ -282,6 +282,43 @@ public String stringValue() {
         }
     }
 
+    /**
+     * String value ignoring the access restrictions.
+     * Non-public fields of an object will also be stringified.
+     *
+     * @return string value
+     */
+    public String stringValueIgnoreAccessRestriction () {
+        readLock.lock();
+        StringJoiner sj = new StringJoiner(", ", "{", "}");
+        try {
+            switch (type.getTag()) {
+                case TypeTags.OBJECT_TYPE_TAG:
+                    for (BField field : ((BStructureType) this.type).getFields()) {
+                        String fieldName = field.getFieldName();
+                        V fieldVal = get((K) fieldName);
+                        sj.add(fieldName + ":" + getStringValue(fieldVal));
+                    }
+                    break;
+                case TypeTags.JSON_TAG:
+                    return getJSONString();
+                default:
+                    String keySeparator = type.getTag() == TypeTags.MAP_TAG ? "\"" : "";
+                    for (Iterator<Map.Entry<K, V>> i = map.entrySet().iterator(); i.hasNext();) {
```

Comment:
Same here. Lets check on whether java 8 streams api can be used to write better code in writing this logic?